runtime/help/commands.md

```diff
@@ -36,6 +36,10 @@ quotes here but these are not necessary when entering the command in micro.
    A negative number can be passed to go inward from the end of the file.
    Example: -5 goes to the 5th-last line in the file.
 
+* `jump 'line[:col]'`: goes with the given relative number from the current
+   line (and optional absolute column) number.
+   Example: -5 goes 5 lines up in the file, while (+)3 goes 3 lines down.
```

Comment:
In this line, `jumps` was actually fine, I think (perhaps even better than `goes`, to make the language more diverse).